qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -102,7 +98,7 @@ public void getPossibleKmerStringTest()  {
 	}
 	
 	@Test
-	public void bothReversedTest() throws IOException {		
+	public void bothReversedTest() throws Exception {		
```

Comment:
are any exceptions thrown from this method? If not, please remove, otherwise be more specific in terms of the type of exception